restful.module

```diff
@@ -426,6 +426,20 @@ function restful_menu_access_callback($resource_name, $version = NULL) {
     return;
   }
 
+  // Set the request and method on the handler, so access callbacks have full
+  // access to the request and account.
```

Comment:
But you're setting them in the resource handler (https://github.com/RESTful-Drupal/restful/blob/token-get/restful.module#L411). As I understand, the `process` method will end up doing this anyways.